src/PaymentGateways/SettingPage.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace  Give\PaymentGateways;
+
+/**
+ * Interface SettingSection
+ * @package Give\Views\Admin\Settings
+ *
+ * @since 2.8.0
+ */
+interface SettingPage {
```

Comment:
Please improve the inline descriptions. Inline docs should help other developers know what to do with something, not merely get an idea of what it does. Contrast the following:
Not too helpful:
```php
	/**
	 * Get section title.
	 * @return string
	 *
	 * @since 2.8.0
	 */
	public function getName();
```
Helpful:
```php
	/**
	 * Provides the section title to be displayed to the user.
	 *
	 * @since 2.8.0
	 *
	 * @return string
	 */
	public function getName();
```

(Also, note the docblock param order.)

Reply:
fixed: https://github.com/impress-org/givewp/compare/324fc09d6f50...4052a9b46690